mission/inspect-train-serve-impl.md

```diff
@@ -4,32 +4,30 @@ First the model needs to be trained using the inspections training
 data. Then one can serve the model to predict the *overall inspection score* and the *lowest
 partial score* based on new data about an inspection.
 
-To take a look at the training in full detail you can find the code [here](). The code for
+To take a look at the training in full detail you can find the code [here](***link missing). The code for
 serving the model is available [here](../src/train/train.py).
  
-In order to increase the quality of the training one first *preprocesses* the data of the inspections.
-
 ## Data Preprocessing
 
 The data preprocessing includes the following steps in sequential order:
  
-1. Scale the targets/labels which in this case are the inspection score and the lowest
+1. Scale the targets and labels, which in this case are the inspection score and the lowest
```

Comment:
target and label are the same int his case, therefore i would stick to targets/labels